cnf-certification-test/performance/suite.go

```diff
@@ -82,3 +90,29 @@ func testSchedulingPolicyInCPUPool(env *provider.TestEnvironment,
 	}
 	tnf.ClaimFilePrintf("Compliant", compliantContainers)
 }
+
+func testRtAppsNoExecProbes(env *provider.TestEnvironment, cuts []*provider.Container) {
+	badContainers := []string{}
+	for _, cut := range cuts {
+		pids, err := crclient.GetContainerPids(cut, env)
+		if err != nil {
+			badContainers = append(badContainers, cut.String())
+			tnf.ClaimFilePrintf("Could not determine the processes pids for container %s, err: %v", cut, err)
+			break
+		}
+		for _, pid := range pids {
+			schedPolicy, _, err := scheduling.GetProcessCPUScheduling(pid, cut)
+			if err != nil {
+				badContainers = append(badContainers, cut.String())
+				tnf.ClaimFilePrintf("Could not determine the scheduling policy for container %s (pid=%v), err: %v", cut, pid, err)
+				break
+			}
+			if scheduling.PolicyIsRT(schedPolicy) && cut.HasExecProbes() {
+				badContainers = append(badContainers, cut.String())
+				tnf.ClaimFilePrintf("Container %s defines exec probes while having a RT scheduling policy", cut)
+				break
```

Comment:
Instead of breaking after finding an offending PID continue testing all PIDs and print a log indicating the pod/container/pid/process_name. Still return  list of non compliant containers.

Reply:
Agree on performing the iteration for all PIDs and logging the result. But I do not think it's a good idea to modify the function `GetContainerPids()` to return the process command as it is fixed for all inputs (apart from the specific PID number) and there's already an error being returned.

Reply:
What do you mean by: "it is fixed for all inputs"? I was just thinking on returning a list of some struct [int, string] but there are many other ways.

Reply:
Ah, I thought you referred to the command used to determine the PID scheduling policy, but I now I see you meant the process name.
I think it's better to add that in another PR as there might be other test cases that could used that improved logging info as well.